### ISSUE TITLE
Modify the behavior of the create resource button

### ISSUE THREAD

**Issue by keithmanville** [usnistgov/dioptra#625]
The create resource button currently opens a drawer with a single option to create the resource for the current page. The behavior should be modified so that it provides an option to create any resource. If the creation is canceled, it should go back to the previous page.

### Blocked by:
- #621

### Definition of done:
- [ ] The behavior of the create resource button is updated
- [ ] This feature is merged into dev
